### PR TITLE
guides for remix, nuxt & vue

### PR DESCRIPTION
**Pull request by Mr-Malomz**
Added guides for the following:

- Keel + Remix
- Keel + Nuxt
- Keel + Vue

### CODE REVIEW

File: pages/keel-nuxt.mdx [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+Finally, create a `components/CreateTaskForm.vue` file to implement task creation using the `createTask` server handler.
+
+```vue
+<script setup lang="ts">
+const description = ref<string>("");
+const errorMsg = ref<string>("");
+const emit = defineEmits();
+
+const onSubmit = async () => {
+    const response = await $fetch("/api/createTask", {
```
Comment (jonbretman) [inline pages/keel-nuxt.mdx:236]: Why not use the Keel client directly here like it's done for listing tasks? Would avoid the need for the api endpoint.

Comment (Mr-Malomz) [reply pages/keel-nuxt.mdx:236]: Yea. We could do that, but my intention here was just to demonstrate how Keel supports usage both on the client and on the server using Nuxt server handlers.

**Comment by tomfrew** [comment]
Let's pop these under guides not misc and give them the titles of "Keel with Next" etc in the navigation

**Comment by Mr-Malomz** [comment]
> Let's pop these under guides not misc and give them the titles of "Keel with Next" etc in the navigation

Done!

File: pages/keel-nuxt.mdx [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+After completing this step, you should have a `nuxt-guide` directory containing the Nuxt project. Now, navigate to that directory with the command below.
+
+```bash
+cd nuxt-guide
+```
+
+## Build your backend with Keel
+
+Keel utilizes a schema-driven approach to application development. This implies that all you need to construct a fully managed backend is to define a [schema](https://docs.keel.so/what-is-keel#schema) comprising data models, APIs, and permissions.
```
Comment (TejasQ) [inline pages/keel-nuxt.mdx:30]: Let’s use a relative link here and everywhere else.

File: pages/keel-remix.mdx [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+Keel utilizes a schema-driven approach to application development. This means that all you need to construct a fully managed backend is to define a [schema](https://docs.keel.so/what-is-keel#schema) comprising data models, APIs, and permissions.
+
+In this guide, you’ll learn to quickly set up a CRUD backend for a basic task manager.
+
+### The schema
+
+Create a new file called `schema.keel` in the project root directory and insert the snippet below:
+
+```protobuf
```
Comment (TejasQ) [inline pages/keel-remix.mdx:39]: ?

```suggestion
```keel
```

**Comment by TejasQ** [comment]
Closing in favor of #92

**Pull request closed without merging**